Tests/ExperimentIntegrationTests/variables.yaml

```diff
@@ -1,3 +1,8 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
 urls:
   stage_server: "https://stage.experimenter.nonprod.dataops.mozgcp.net"
   prod_server: "https://experimenter.services.mozilla.com"
+  telemetry_server: "http://10.10.1.198:5000" # must be changed
```

Comment:
Nit: must be changed before merging or when testing locally?

Reply:
This will need to be changed when testing locally.